Dockerfile

```diff
@@ -1,72 +1,48 @@
 FROM ubuntu
 USER root
 
-WORKDIR /home
-
-VOLUME ["/asn1_codec"]
-
-RUN export LD_LIBRARY_PATH=/usr/local/lib
-RUN export CC=gcc
+WORKDIR /asn1-codec
 
 # Add build tools.
 RUN apt-get update && apt-get install -y software-properties-common wget git make gcc-4.9 g++-4.9 gcc-4.9-base && update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-4.9 100 && update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.9 100
 
-#install editors vim and nano
-RUN apt-get update && apt-get install -y vim
```

Comment:
Please do not remove these editors. They are often needed when debugging issues from within the container. Keep in mind that running these applications inside docker container is now the primary mode of deployment, even for WYDOT.